<Type Name="NotFiniteNumberException" FullName="System.NotFiniteNumberException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="e0ed0b79436e1528638f17d00ffb4756a66ceb5a" />
    <Meta Name="ms.sourcegitcommit" Value="d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="es-ES" />
    <Meta Name="ms.lasthandoff" Value="04/03/2018" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class NotFiniteNumberException : ArithmeticException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit NotFiniteNumberException extends System.ArithmeticException" />
  <TypeSignature Language="DocId" Value="T:System.NotFiniteNumberException" />
  <TypeSignature Language="VB.NET" Value="Public Class NotFiniteNumberException&#xA;Inherits ArithmeticException" />
  <TypeSignature Language="C++ CLI" Value="public ref class NotFiniteNumberException : ArithmeticException" />
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.ArithmeticException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary>
      <span data-ttu-id="67f42-101">Excepción que se produce cuando un valor de punto flotante es infinito positivo, infinito negativo o un elemento no numérico (NaN).</span>
      <span class="sxs-lookup">
        <span data-stu-id="67f42-101">The exception that is thrown when a floating-point value is positive infinity, negative infinity, or Not-a-Number (NaN).</span>
      </span>
    </summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-102"><xref:System.NotFiniteNumberException> está disponible para los lenguajes de programación que no son compatibles con los conceptos de infinito y no un número de operaciones de punto flotante.</span><span class="sxs-lookup"><span data-stu-id="67f42-102"><xref:System.NotFiniteNumberException> is available for programming languages that do not support the concepts of infinity and Not-a-Number in floating-point operations.</span></span>  
  
 <span data-ttu-id="67f42-103"><xref:System.NotFiniteNumberException> utiliza HRESULT COR_E_NOTFINITENUMBER, que tiene el valor 0 x 80131528.</span><span class="sxs-lookup"><span data-stu-id="67f42-103"><xref:System.NotFiniteNumberException> uses the HRESULT COR_E_NOTFINITENUMBER, which has the value 0x80131528.</span></span>  
  
 <span data-ttu-id="67f42-104">Para obtener una lista de valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>, consulte el <xref:System.NotFiniteNumberException.%23ctor%2A> constructores.</span><span class="sxs-lookup"><span data-stu-id="67f42-104">For a list of initial property values for an instance of <xref:System.NotFiniteNumberException>, see the <xref:System.NotFiniteNumberException.%23ctor%2A> constructors.</span></span>  
  
 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
    <altmember cref="T:System.Double" />
    <altmember cref="T:System.Single" />
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="67f42-105">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-105">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="67f42-106">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-106">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-107">Este constructor inicializa la <xref:System.Exception.Message%2A> propiedad de la nueva instancia en un mensaje proporcionado por el sistema que describe el error, como "el número encontrado no es una cantidad finita".</span><span class="sxs-lookup"><span data-stu-id="67f42-107">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as "The number encountered was not a finite quantity."</span></span> <span data-ttu-id="67f42-108">Este mensaje tiene en cuenta la referencia cultural del sistema actual.</span><span class="sxs-lookup"><span data-stu-id="67f42-108">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="67f42-109">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>.</span><span class="sxs-lookup"><span data-stu-id="67f42-109">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="67f42-110">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="67f42-110">Property</span></span>|<span data-ttu-id="67f42-111">Valor</span><span class="sxs-lookup"><span data-stu-id="67f42-111">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="67f42-112">Referencia nula (`Nothing` en Visual Basic).</span><span class="sxs-lookup"><span data-stu-id="67f42-112">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="67f42-113">La cadena del mensaje de error localizado.</span><span class="sxs-lookup"><span data-stu-id="67f42-113">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (double offendingNumber);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(float64 offendingNumber) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.Double)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (offendingNumber As Double)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(double offendingNumber);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="offendingNumber" Type="System.Double" />
      </Parameters>
      <Docs>
        <param name="offendingNumber">
          <span data-ttu-id="67f42-114">Valor del argumento que causó la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-114">The value of the argument that caused the exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-115">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" /> con el número no válido.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-115">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with the invalid number.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-116">El `offendingNumber` parámetro debe ser un <xref:System.Double> y un número no válido.</span><span class="sxs-lookup"><span data-stu-id="67f42-116">The `offendingNumber` parameter must be both a <xref:System.Double> and an invalid number.</span></span>  
  
 <span data-ttu-id="67f42-117">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>.</span><span class="sxs-lookup"><span data-stu-id="67f42-117">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="67f42-118">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="67f42-118">Property</span></span>|<span data-ttu-id="67f42-119">Valor</span><span class="sxs-lookup"><span data-stu-id="67f42-119">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="67f42-120">Referencia nula (`Nothing` en Visual Basic).</span><span class="sxs-lookup"><span data-stu-id="67f42-120">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.NotFiniteNumberException.OffendingNumber%2A>|<span data-ttu-id="67f42-121">Número no válido.</span><span class="sxs-lookup"><span data-stu-id="67f42-121">The invalid number.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="67f42-122">Mensaje que describe el error.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-122">The message that describes the error.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-123">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" /> con el mensaje de error especificado.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-123">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-124">Asegúrese de que el `message` cadena es comprensible para el usuario y que se adapta a la referencia cultural actual.</span><span class="sxs-lookup"><span data-stu-id="67f42-124">Ensure that the `message` string is understandable to the user and that it is localized for the current culture.</span></span>  
  
 <span data-ttu-id="67f42-125">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>.</span><span class="sxs-lookup"><span data-stu-id="67f42-125">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="67f42-126">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="67f42-126">Property</span></span>|<span data-ttu-id="67f42-127">Valor</span><span class="sxs-lookup"><span data-stu-id="67f42-127">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="67f42-128">Referencia nula (`Nothing` en Visual Basic).</span><span class="sxs-lookup"><span data-stu-id="67f42-128">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="67f42-129">Cadena con el mensaje de error.</span><span class="sxs-lookup"><span data-stu-id="67f42-129">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected NotFiniteNumberException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; NotFiniteNumberException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <span data-ttu-id="67f42-130">Objeto que contiene los datos del objeto serializados.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-130">The object that holds the serialized object data.</span>
          </span>
        </param>
        <param name="context">
          <span data-ttu-id="67f42-131">Información contextual acerca del origen o del destino.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-131">The contextual information about the source or destination.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-132">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" /> con datos serializados.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-132">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with serialized data.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-133">Durante la deserialización se llama a este constructor para reconstituir el objeto de excepción transmitido en una secuencia.</span><span class="sxs-lookup"><span data-stu-id="67f42-133">This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message, double offendingNumber);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, float64 offendingNumber) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String,System.Double)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, offendingNumber As Double)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message, double offendingNumber);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="offendingNumber" Type="System.Double" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="67f42-134">Mensaje que describe el error.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-134">The message that describes the error.</span>
          </span>
        </param>
        <param name="offendingNumber">
          <span data-ttu-id="67f42-135">Valor del argumento que causó la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-135">The value of the argument that caused the exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-136">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" /> con el mensaje de error especificado y el número no válido.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-136">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message and the invalid number.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-137">Asegúrese de que el `message` cadena es comprensible para el usuario y que se adapta a la referencia cultural actual.</span><span class="sxs-lookup"><span data-stu-id="67f42-137">Ensure that the `message` string is understandable to the user and that it is localized for the current culture.</span></span>  
  
 <span data-ttu-id="67f42-138">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>.</span><span class="sxs-lookup"><span data-stu-id="67f42-138">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="67f42-139">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="67f42-139">Property</span></span>|<span data-ttu-id="67f42-140">Valor</span><span class="sxs-lookup"><span data-stu-id="67f42-140">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="67f42-141">Referencia nula (`Nothing` en Visual Basic).</span><span class="sxs-lookup"><span data-stu-id="67f42-141">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="67f42-142">Cadena con el mensaje de error.</span><span class="sxs-lookup"><span data-stu-id="67f42-142">The error message string.</span></span>|  
|<xref:System.NotFiniteNumberException.OffendingNumber%2A>|<span data-ttu-id="67f42-143">Número no válido.</span><span class="sxs-lookup"><span data-stu-id="67f42-143">The invalid number.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="67f42-144">Mensaje de error que explica el motivo de la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-144">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="67f42-145">Excepción que es la causa de la excepción actual.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-145">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="67f42-146">Si el parámetro <c>innerException</c> no es una referencia nula (<see langword="Nothing" /> en Visual Basic), la excepción actual se inicia en un bloque <see langword="catch" /> que controla la excepción interna.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-146">If the <c>innerException</c> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-147">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" /> con un mensaje de error especificado y una referencia a la excepción interna que representa la causa de esta excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-147">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message and a reference to the inner exception that is root cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-148">Una excepción que se produce como resultado directo de una excepción anterior puede incluir una referencia a la excepción anterior de la propiedad <xref:System.Exception.InnerException%2A>.</span><span class="sxs-lookup"><span data-stu-id="67f42-148">An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="67f42-149">La propiedad <xref:System.Exception.InnerException%2A> devuelve el mismo valor que se haya pasado al constructor, o bien `null` en caso de que la propiedad <xref:System.Exception.InnerException%2A> no le haya proporcionado al constructor el valor de la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="67f42-149">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="67f42-150">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>.</span><span class="sxs-lookup"><span data-stu-id="67f42-150">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="67f42-151">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="67f42-151">Property</span></span>|<span data-ttu-id="67f42-152">Valor</span><span class="sxs-lookup"><span data-stu-id="67f42-152">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="67f42-153">Referencia a la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="67f42-153">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="67f42-154">Cadena con el mensaje de error.</span><span class="sxs-lookup"><span data-stu-id="67f42-154">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message, double offendingNumber, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, float64 offendingNumber, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String,System.Double,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, offendingNumber As Double, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message, double offendingNumber, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="offendingNumber" Type="System.Double" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="67f42-155">Mensaje de error que explica el motivo de la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-155">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <param name="offendingNumber">
          <span data-ttu-id="67f42-156">Valor del argumento que causó la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-156">The value of the argument that caused the exception.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="67f42-157">Excepción que es la causa de la excepción actual.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-157">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="67f42-158">Si el parámetro <c>innerException</c> no es una referencia nula (<see langword="Nothing" /> en Visual Basic), la excepción actual se inicia en un bloque <see langword="catch" /> que controla la excepción interna.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-158">If the <c>innerException</c> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-159">Inicializa una nueva instancia de la clase <see cref="T:System.NotFiniteNumberException" /> con un mensaje de error especificado, el número no válido y una referencia a la excepción interna que representa la causa principal de esta excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-159">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message, the invalid number, and a reference to the inner exception that is root cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-160">Una excepción que se produce como resultado directo de una excepción anterior puede incluir una referencia a la excepción anterior de la propiedad <xref:System.Exception.InnerException%2A>.</span><span class="sxs-lookup"><span data-stu-id="67f42-160">An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="67f42-161">La propiedad <xref:System.Exception.InnerException%2A> devuelve el mismo valor que se haya pasado al constructor, o bien `null` en caso de que la propiedad <xref:System.Exception.InnerException%2A> no le haya proporcionado al constructor el valor de la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="67f42-161">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="67f42-162">En la tabla siguiente se muestran los valores de propiedad iniciales para una instancia de <xref:System.NotFiniteNumberException>.</span><span class="sxs-lookup"><span data-stu-id="67f42-162">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="67f42-163">Propiedad.</span><span class="sxs-lookup"><span data-stu-id="67f42-163">Property</span></span>|<span data-ttu-id="67f42-164">Valor</span><span class="sxs-lookup"><span data-stu-id="67f42-164">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="67f42-165">Referencia a la excepción interna.</span><span class="sxs-lookup"><span data-stu-id="67f42-165">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="67f42-166">Cadena con el mensaje de error.</span><span class="sxs-lookup"><span data-stu-id="67f42-166">The error message string.</span></span>|  
|<xref:System.NotFiniteNumberException.OffendingNumber%2A>|<span data-ttu-id="67f42-167">Número no válido.</span><span class="sxs-lookup"><span data-stu-id="67f42-167">The invalid number.</span></span>|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
    <Member MemberName="GetObjectData">
      <MemberSignature Language="C#" Value="public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig virtual instance void GetObjectData(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Public Overrides Sub GetObjectData (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; override void GetObjectData(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Method</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute>
          <AttributeName>System.Security.SecurityCritical</AttributeName>
        </Attribute>
      </Attributes>
      <ReturnValue>
        <ReturnType>System.Void</ReturnType>
      </ReturnValue>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <span data-ttu-id="67f42-168">Objeto que contiene los datos del objeto serializados.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-168">The object that holds the serialized object data.</span>
          </span>
        </param>
        <param name="context">
          <span data-ttu-id="67f42-169">Información contextual acerca del origen o del destino.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-169">The contextual information about the source or destination.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="67f42-170">Establece el objeto <see cref="T:System.Runtime.Serialization.SerializationInfo" /> con el número no válido y la información adicional de la excepción.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-170">Sets the <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object with the invalid number and additional exception information.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-171"><xref:System.TypeLoadException.GetObjectData%2A> establece <xref:System.Runtime.Serialization.SerializationInfo> con todos los datos del objeto de excepción destinados a la serialización.</span><span class="sxs-lookup"><span data-stu-id="67f42-171"><xref:System.TypeLoadException.GetObjectData%2A> sets a <xref:System.Runtime.Serialization.SerializationInfo> with all the exception object data targeted for serialization.</span></span> <span data-ttu-id="67f42-172">Durante la deserialización, el objeto de excepción se reconstituye a partir del <xref:System.Runtime.Serialization.SerializationInfo> transmitida en la secuencia.</span><span class="sxs-lookup"><span data-stu-id="67f42-172">During deserialization, the exception object is reconstituted from the <xref:System.Runtime.Serialization.SerializationInfo> transmitted over the stream.</span></span>  
  
 <span data-ttu-id="67f42-173">Para obtener más información, vea <xref:System.Runtime.Serialization.SerializationInfo>.</span><span class="sxs-lookup"><span data-stu-id="67f42-173">For more information, see <xref:System.Runtime.Serialization.SerializationInfo>.</span></span>  
  
 ]]></format>
        </remarks>
        <exception cref="T:System.ArgumentNullException">
          <span data-ttu-id="67f42-174">El objeto <paramref name="info" /> es <see langword="null" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-174">The <paramref name="info" /> object is <see langword="null" />.</span>
          </span>
        </exception>
        <permission cref="T:System.Security.SecurityCriticalAttribute">
          <span data-ttu-id="67f42-175">requiere plena confianza para el llamador inmediato.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-175">requires full trust for the immediate caller.</span>
          </span>
          <span data-ttu-id="67f42-176">Este miembro no puede usarse por código de confianza parcial o transparente.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-176">This member cannot be used by partially trusted or transparent code.</span>
          </span>
        </permission>
        <altmember cref="T:System.Runtime.Serialization.SerializationInfo" />
        <altmember cref="T:System.Runtime.Serialization.StreamingContext" />
      </Docs>
    </Member>
    <Member MemberName="OffendingNumber">
      <MemberSignature Language="C#" Value="public double OffendingNumber { get; }" />
      <MemberSignature Language="ILAsm" Value=".property instance float64 OffendingNumber" />
      <MemberSignature Language="DocId" Value="P:System.NotFiniteNumberException.OffendingNumber" />
      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property OffendingNumber As Double" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; property double OffendingNumber { double get(); };" />
      <MemberType>Property</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <ReturnValue>
        <ReturnType>System.Double</ReturnType>
      </ReturnValue>
      <Docs>
        <summary>
          <span data-ttu-id="67f42-177">Obtiene el número no válido que es infinito positivo, infinito negativo o un elemento no numérico (NaN).</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-177">Gets the invalid number that is a positive infinity, a negative infinity, or Not-a-Number (NaN).</span>
          </span>
        </summary>
        <value>
          <span data-ttu-id="67f42-178">Número no válido.</span>
          <span class="sxs-lookup">
            <span data-stu-id="67f42-178">The invalid number.</span>
          </span>
        </value>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="67f42-179">El <xref:System.NotFiniteNumberException.OffendingNumber%2A> propiedad devuelve el mismo valor que se pasó al constructor, o cero si no hay ningún valor se ha proporcionado.</span><span class="sxs-lookup"><span data-stu-id="67f42-179">The <xref:System.NotFiniteNumberException.OffendingNumber%2A> property returns the same value as was passed into the constructor, or zero if no value was supplied.</span></span> <span data-ttu-id="67f42-180">Invalidar el <xref:System.NotFiniteNumberException.OffendingNumber%2A> propiedad para personalizar el contenido o el formato del valor no válido.</span><span class="sxs-lookup"><span data-stu-id="67f42-180">Override the <xref:System.NotFiniteNumberException.OffendingNumber%2A> property to customize the content or format of the invalid value.</span></span>  
  
 <span data-ttu-id="67f42-181">Esta propiedad es de sólo lectura.</span><span class="sxs-lookup"><span data-stu-id="67f42-181">This property is read-only.</span></span>  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Double" />
      </Docs>
    </Member>
  </Members>
</Type>